src/atoms/CustomHtml/CustomHtml.js

```diff
@@ -1,34 +1,99 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
 import whitelist from '../../utils/whitelist.json';
 
 class CustomHtml extends React.Component {
   constructor(props) {
     super(props);
-
+    this.myRef = React.createRef();
     this.state = {
-      html: ''
+      html: '',
+      loaded: false
     };
   }
 
   componentDidMount() {
+    const { html, safeScripts } = this.parseHtml({
+      ...this.props.nodeData.attrs,
+      whitelist: whitelist.approved
+    });
+
+    // eslint-disable-next-line
+    const node = ReactDOM.findDOMNode(this.myRef.current);
```

Comment:
Not groking why myRef isn't just empty at this point

Reply:
I followed these docs: https://reactjs.org/docs/refs-and-the-dom.html . I assume it isn't empty because when the component has mounted there should be a node? I verified it works though because If you examine the code running the scripts get attached to their own component's DOM customHtml block.

Reply:
ok thanks for explaining